sound/soc/sof/Kconfig

```diff
@@ -36,6 +36,14 @@ config SND_SOC_SOF_ACPI
 	  Say Y if you need this option
 	  If unsure select "N".
 
+config SND_SOC_SOF_DT
+	tristate "SOF DT enumeration support"
+	select SND_SOC_SOF
+	help
+	  This adds support for Device Tree enumeration. This option is
+	  required to enable i.MX8 devices.
+	  Say Y if you need this option. If unsure select "N".
+
```

Comment:
this should probably the squashed with the earlier addition of the top-level Kconfig otherwise you are going to have broken configs?